cirq/google/common_serializers.py

```diff
@@ -437,7 +459,7 @@ def _convert_physical_z(op: ops.Operation, proto: v2.program_pb2.Operation):
     op_serializer.GateOpSerializer(
         gate_type=ops.FSimGate,
         serialized_gate_id='fsim_pi_4',
-        args=[],
+        args=[_phase_match_arg],
```

Comment:
This is a little confusing, since the default is to match via physical Z for sqrt iswap, correct?

Reply:
That's correct. I think it's nice to have this here for consistency with other gates. I've added a note about this in the PR description.